src/common/templates/_dbv3.tpl

```diff
@@ -701,6 +701,7 @@ USAGE:
     {{- if .localTimescaleDBCtx }}
         {{- $localTimescaleDBCtx = .localTimescaleDBCtx }}
     {{- end }}
+    {{- $database := default .database $localTimescaleDBCtx.database }}
```

Comment:
do we need to make this change in `harnesscommon.dbconnectionv4.timescaleConnection` as well ?

Reply:
Not sure if anyone uses that.